### ISSUE TITLE
Timepicker A11y tags are hardcoded to English - needs i18n

### ISSUE THREAD

**Issue by mendeza** [ng-bootstrap/ng-bootstrap#2119]
### Bug description:
In the Timepicker template, `sr-only` text content and `aria-label` values are not internationalized; they are hardcoded to English. These should be localizable.

### Link to the code
https://github.com/ng-bootstrap/ng-bootstrap/blob/master/src/timepicker/timepicker.ts
E.g., line 73
```<span class="sr-only">Increment hours</span>```
E.g., line 77
```aria-label="Hours"```

### Version of Angular, ng-bootstrap, and Bootstrap:

Angular: N/A

ng-bootstrap: v1.0.0-beta.9

Bootstrap: N/A

**Comment by pkozlowski-opensource** [comment]
Normally those things should be translated using the standard Angular i18n / translation mechanism, as there you should access to all the templates (including ones from this library). 

Are ng-bootstrap templates not part of the translation or are you using something else that standard Angular translation pipeline?

**Comment by pkozlowski-opensource** [comment]
Closing since I haven't heard back...
